docs/website/root/mithril/threat-model.md

```diff
@@ -0,0 +1,293 @@
+- Also we should update the [protocol page](https://mithril.network/doc/mithril/mithril-protocol/protocol)
+
+:::
+
+System consists of three main components: signers, aggregator and clients.
+
+Mithril signers do certify Cardano chain data using a mithril signing key. That means, mithril signers need access to a trusted `cardano-node` and the mithril signing key to operate.
+
+Mithril signing keys are rotated every epoch and need to be certified by the Cardano KES key. For this, the mithril signers need access to the KES key in order to register the signing key for this epoch. 
+
+Cardano KES keys are also used block producing `cardano-node` and are typically located on the block producing machine.
+
+Cardano KES keys need to be [evolved every 36 hours](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/stake-pool-operations/7_KES_period.md), while they can be rotated from a root key when needed.
+
+:::info To do
+
+Is there a Cardano threat model about this?
+
+:::
+
+All mithril signers and mithril clients connect to a single aggregator using HTTP over TLS.
+
+Registering a Mithril signing key means that a signer sends its corresponding verification key to the aggregator, for the purpose of distribution to all other mithril signers.
+
+A mithril aggregator coordinates creation of signatures by all registered signers. Mithril signers do ask the aggregater whether a signature is pending on a regular basis. The aggregator responds with information what to sign and a list of public information about all registered signers.
+
+Each mithril signer verifies the information, produces a signature of the requested information to sign and submits that to the aggregator (which verifies the signature being correct upon receiving).
+
+The aggregator repeatedly checks whether enough valid signatures (to reach the quorum) are available to aggregate a Mithril stake-based multi-signature into a certificate.
+
+Mithril certificates are certifying some chain data using an aggregated multi-signature verification key and are chained up to some genesis certificate, which is signed by a genesis signing key.
+
+Mithril clients do connect to an aggregator using HTTP over TLS to query mithril certificates for certified chain data and locate artifacts.
+
+A mithril client can verify the received mithril certificate is linked to other certificates up to the genesis certificate and can be verified using the Mithril genesis verification key (see [details](https://mithril.network/doc/mithril/mithril-protocol/certificates/)).
+
+:::info To do
+
+Missing: the currently recommended relay (reverse proxy)
+
+:::
+
+#### Deployment architecture
+
+This document is specifically targeting the standard deployment architecture where a mithril signer runs next to the block producing node, while access to the mithril aggregator is only done through a relay.
+
+[![Mithril - Container View](./images/deployment-architecture.jpg)](./images/deployment-architecture.jpg)
```

Comment:
The schema is referring to the experimental P2P relay implementation which is not deployed to production.
Maybe a link to these docs is enough:
-  [architecture](https://mithril.network/doc/mithril/mithril-network/architecture#architecture-diagram) 
- [deployment model](https://mithril.network/doc/manual/getting-started/run-signer-node#mithril-signer-deployment-model) in the SPO guide

WDYT?